src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -89,6 +97,23 @@ public void after() throws Exception {
         super.after();
     }
 
+    public void getUserToken(boolean isAdmin) {
```

Comment:
Nit: This method says `get...` but doesn't return anything. Consider either returning the JWT instead of assigning it to a class field, or renaming the method to something like `setupUserToken`.